sklearn/model_selection/_search.py

```diff
@@ -897,6 +897,10 @@ def _format_results(self, candidate_params, n_splits, out,
         out = _aggregate_score_dicts(out)
 
         results = dict(more_results or {})
+        for key, val in results.items():
+            # each value is a list (as per evaluate_candidate's convention)
+            # we convert it to an array for consistency with the other keys
+            results[key] = np.asarray(val)
```

Comment:
I updated this to use `asarray` so we do not make a copy if `val` is already an ndarray.